met-web/src/components/Form/FormBuilder.tsx

```diff
@@ -6,12 +6,14 @@ import { FormBuilderData, FormBuilderProps } from './types';
 
 const FormBuilder = ({ handleFormChange, savedForm }: FormBuilderProps) => {
     return (
-        <FormioFormBuilder
-            form={savedForm || { display: 'form' }}
-            options={formioOptions}
-            saveText={'Create Form'}
-            onChange={(form: unknown) => handleFormChange(form as FormBuilderData)}
-        />
+        <div className="formio">
+            <FormioFormBuilder
+                form={savedForm || { display: 'form' }}
+                options={formioOptions}
+                saveText={'Create Form'}
+                onChange={(form: unknown) => handleFormChange(form as FormBuilderData)}
+            />
+        </div>
     );
```

Comment:
I think add a comment saying why we went for this approach since a new dev might not understand just by looking at the code when an extra div is placed... this is worth explaining via comments